hibernate-core/src/main/java/org/hibernate/tool/schema/internal/ColumnDefinitions.java

```diff
@@ -248,6 +264,8 @@ private static String normalize(String typeName) {
 		else {
 			final String lowerCaseTypName = typeName.toLowerCase(Locale.ROOT);
 			switch (lowerCaseTypName) {
+				case "int":
+					return "integer";
```

Comment:
This looks correct, but which database is it required for?

Reply:
MySQL return `int` for `ResultSetMetaData::getColumnTypeName` even `columnDefinition` is `integer`.

Reply:
OK, thanks.